.vscode/settings.json

```diff
@@ -6,12 +6,11 @@
   "editor.inlineSuggest.enabled": true,
   "editor.insertSpaces": true,
   "editor.minimap.enabled": true,
-  "editor.rulers": [80, 100],
+  "editor.rulers": [110],
```

Comment:
**question**: why is this changed? It will cause the files will format and we will have changes related to code styling instead of the intentional change.